test/Microsoft.AspNetCore.Routing.Tests/InlineRouteParameterParserTests.cs

```diff
@@ -29,7 +29,7 @@ public void ParseRouteParameter_WithoutADefaultValue(string parameterName)
         }
 
         [Fact]
-        public void ParseRouteParameter_WithoutADefaultValue()
+        public void ParseRouteParameter_WithEmptyDefaultValue()
```

Comment:
This is a known issue with xunit that they have decided not to fix. https://github.com/xunit/xunit/issues/650